crates/fj-viewer/src/gui.rs

```diff
@@ -348,9 +353,13 @@ impl Gui {
 }
 
 fn show_file_dialog() -> Option<PathBuf> {
-    FileDialog::new()
+    #[cfg(not(target_arch = "wasm32"))]
+    return FileDialog::new()
         .set_directory(current_dir().unwrap_or_else(|_| PathBuf::from("/")))
-        .pick_folder()
+        .pick_folder();
+
+    #[cfg(target_arch = "wasm32")]
+    todo!("Picking folders does not work on wasm32")
```

Comment:
This (and the other `cfg`s) aren't great, but I can't think of a better way to do it right now. Maybe the file picking feature should move out of `fj-viewer` and into `fj-window` or `fj-app`? No idea. We'll figure it out at some point!